PROJECT_PROGRESSION.md

```diff
@@ -16,14 +16,60 @@ This governance policy sets forth the proposal process for projects to be accept
 
 New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.
 
-#### Application Phases
+#### Roles
+
+##### Applicant
+
+Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.
```

Comment:
How do we verify that an applicant can represent the project in question?

Reply:
Do we need to verify this? Isn't it simply another form of a Developer Certificate of Origin (DCO)?

Reply:
We have that later in the fit checklist. We purposely left if vague as to how we'd validate that.